github/ci/services/grafana/hack/deploy.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -eo pipefail
+
+main() {
+	local environment=${1}
+
+	bazelisk run //github/ci/services/grafana:$environment.apply
+
+	bazelisk run //github/ci/services/common/k8s/cmd/wait -- -namespace monitoring -selector grafana-deployment -kind deployment
+
+	# Allow some time for the grafana operator to create the required pods & services
+	sleep 10
```

Comment:
This is pretty brittle, no? Shouldn't we use something like a pod readiness check or maybe even the cr state? 

Note: I've tried this one and it seemed to work
```bash
kubectl wait --for=jsonpath='{.status.stage}'=complete -n openshift-operators grafana grafana
```